script.js

```diff
@@ -0,0 +1,45 @@
+// Set variables for each respective move
+var x = "X"
+var o = "O"
+// Create functions to place X or O, taking in boxNumber to choose where to place
+function placeX(boxNumber) {
+	document.getElementById(boxNumber).innerHTML = x;
+}
+function placeO(boxNumber) {
+	document.getElementById(boxNumber).innerHTML = o;
+}
+
+// Set variables to easily select individual cells and determine which player's turn it is
+
+var cells = document.getElementsByTagName("TD");
+var playerTurn = document.getElementsByClassName("player-turn")[0];
+
+
+var moveCounter = 1;
+playerTurn.innerHTML = "X's turn!"
+function makeMove(boxNumber) {
+	moveCounter++;
+	if(cells[boxNumber-1].innerHTML != "X" || "O") {
+		if (moveCounter % 2 === 0) {
+			placeO(boxNumber);
+			playerTurn.innerHTML = "X's turn!"
+		} else {
+			placeX(boxNumber);
+			playerTurn.innerHTML = "O's turn!"
+		}
+	} else {
+		console.log("Place somewhere else.");
+	}
+}
+
+
+// Reset board
+var reset = document.getElementsByTagName("A")[0];
+
+reset.addEventListener("click", resetBoard);
+
+function resetBoard() {
+for(i = 0; i <= 8; i++) {
```

Comment:
Never use a constant in a for loop! instead of `i <= 8`, it should be `i <= array.length`